spec/001_song_basics_spec.rb

```diff
@@ -1,54 +1,68 @@
-require 'spec_helper'
+require "spec_helper"
 
-describe "Song Basics" do
-  describe '#initialize with #name' do
-    it 'accepts a name for the song' do
-      song = Song.new("In the Aeroplane Over the Sea")
+describe "Song" do
+  let(:song) { Song.new("In the Aeroplane Over the Sea") }
```

Comment:
Ended up keeping the `let`s but overriding them for the first couple tests (until the student has a functioning `#initialize` method) so that the error messages never mention the `let`s.